playbooks/foreman.yml

```diff
@@ -0,0 +1,8 @@
+- hosts: all
```

Comment:
I renamed katello_nightly.yml to katello.yml during the puppet 4 work, so this keeps that convention. I actually meant to delete the existing foreman_nightly.yml.

The reason for that was the version is really controlled through the repositories installed.  And now since you can set host vars in the boxes.yaml, you can do this:

```
myforeman:
  box: centos7
  memory: 8096
  ansible:
    playbook:
        - 'playbooks/foreman.yml'
    group: 'server'
    variables:
      foreman_version: 1.11
```

So version-specific playbook doesn't make sense IMHO